src/dotnet/MLSearch.Service/Engine/SearchQuery.cs

```diff
@@ -1,21 +1,21 @@
 
 namespace ActualChat.MLSearch.Engine;
 
-internal sealed class SearchQuery
+public sealed class SearchQuery
```

Comment:
Why has everything turned to public? Is this avoidable?

Reply:
Let me state the reason first:
services.AddMvcCore().AddApplicationPart(GetType().Assembly);
Apparently this method scans for public controllers only. It silently ignores all controllers declared internal.
It is possible to declare public controller classes with internal constructors and skip primary constructors all together. Not sure what is better here. Will discuss offline